web/components/NavBlock/NavBlock.tsx

```diff
@@ -2,102 +2,68 @@ import Link from 'next/link';
 import clsx from 'clsx';
 import styles from './NavBlock.module.css';
 
+interface FakeItemProps {
+  divider?: boolean;
+  mobile?: boolean;
+  tablet?: boolean;
+  desktop?: boolean;
+}
+
+const FakeItem = ({ divider, mobile, tablet, desktop }: FakeItemProps) =>
+  <li
+    className={clsx(
+     divider ? styles.divider : styles.fakeItem,
+      mobile && styles.mobile,
+      tablet && styles.tablet,
+      desktop && styles.desktop
+    )}
+    aria-hidden="true"
+  />
+
 export const NavBlock = () => (
   <nav className={styles.nav}>
     <ul className={styles.list}>
+      <FakeItem mobile />
       <li className={styles.item}>
```

Comment:
Nit: There are empty lines around every real link (spacing them apart from the `FakeItem`s), so for consistency there could be one here as well.

Reply:
Good. 27f6040.